LoopMusic/Source/MusicPlayer/LoopScrubber.swift

```diff
@@ -55,7 +62,25 @@ class LoopScrubber: UISlider {
     /// Starts the slider update loop.
     func startTimer() {
         if self.playbackTimer == nil {
-            self.playbackTimer = Timer.scheduledTimer(withTimeInterval: LoopScrubber.self.PLAYBACK_TIMER_INTERVAL, repeats: true) { [weak self] _ in self?.updateValue() }
+            if fastMode {
+                self.playbackTimer = Timer.scheduledTimer(withTimeInterval: LoopScrubber.self.PLAYBACK_TIMER_INTERVAL_FAST, repeats: true) { [weak self] _ in
+                    self?.updateValue()
+                    if !MusicPlayer.player.loopPlayback || MusicPlayer.player.loopEndSeconds - MusicPlayer.player.playbackTimeSeconds > LoopScrubber.INTERVAL_THRESHOLD*LoopScrubber.PLAYBACK_TIMER_INTERVAL {
```

Comment:
This is a decently long boolean expression that could be extracted into a helper function to avoid duplication.